contessa/consistency_checker.py

```diff
@@ -111,22 +111,9 @@ def run_query(self, table_name: str, conn: Connector, method: str):
         result = [r for r in conn.get_records(query)]
         return result
 
-    def insert(self, dc_cls, result):
-        """
```

Comment:
Let's not loose comment please :) Can you maybe shortly describe the consequence - that the results will be overwritten, etc.?

Reply:
actually. i added comment to the `upsert` method of connector, to avoid having bad comments on multiple places if we rewrite it (its in both runners)

Reply:
I know the `upsert` term should be self-explanatory, I meant provide some general comment, e.g. reason why doing upsert at all